pkg/controller/smart-clone-controller.go

```diff
@@ -326,7 +330,6 @@ func newPvcFromSnapshot(snapshot *snapshotv1.VolumeSnapshot, targetPvcSpec *core
 			},
 			Annotations: map[string]string{
 				AnnSmartCloneRequest:       "true",
-				AnnCloneOf:                 "true",
```

Comment:
This was issue #1.  That annotation should only be added when PVC is bound.  Because the datavolume controller will initiate transfer based on that